cmd/config/config.go

```diff
@@ -21,6 +21,12 @@ type Config struct {
 	Project string `json:"project" pflag:",Specifies the project to work on."`
 	Domain  string `json:"domain" pflag:",Specified the domain to work on."`
 	Output  string `json:"output" pflag:",Specified the output type."`
+	Filename  string `json:"filename" pflag:",Specified the filename."`
```

Comment:
i dont think these should be added at the global level. we should create new configs for each command and they will automatically get added using pflags generator
 https://github.com/lyft/flytestdlib/tree/master/cli/pflags

Example: https://github.com/lyft/flytepropeller/blob/master/pkg/controller/config/config.go